Automattic/wp-calypso
# Happy Blocks: Remove support content footer DIFM banner

Related to p1717704306331939-slack-C02T4NVL4JJ

## Proposed Changes

* Removes the DIFM CTA in the `support-content-footer` block.

<img width="1328" alt="CleanShot 2024-06-06 at 23 46 55@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/3dc135bd-0441-4629-8a61-19dcd32da1ec">


## Why are these changes being made?


*

## Testing Instructions



* Checkout branch
* Sandbox public.api and wordpress.com
* Navigate to `/apps/happy-blocks` and run `yarn dev --sync` to sync to sandbox
* Navigate to wordpress.com/support, scroll to the bottom of the page, and verify the difm CTA banner has been removed.

## Pre-merge Checklist
